bodhi/tests/server/services/test_releases.py

```diff
@@ -226,6 +226,32 @@ def test_list_releases_with_a_wrong_state(self):
         # so let's just make sure we have an error.
         self.assertEqual(body['status'], 'error')
 
+    def test_list_releases_exclude_archived_on(self):
+        """ Test that we can filter releases using the exclude_archived flag"""
+        r = Release.query.filter_by(name='F17').first()
+        r.state = ReleaseState.archived
+        r = Release.query.filter_by(name='F22').first()
+        r.state = ReleaseState.current
+        self.db.flush()
+        res = self.app.get('/releases/', {"exclude_archived": True})
+        body = res.json_body
+        self.assertEqual(len(body['releases']), 1)
```

Comment:
We can make this assertion a little stronger:

```suggestion
        self.assertEqual([r['name'] for r in body['releases']], ['F22'])
```